e2e/ansible/playbooks/hyperconverged/test-k8s-percona-mysql-pod/k8s-percona-pod-prerequisites.yml

```diff
@@ -29,6 +29,14 @@
   delegate_to: "{{ groups['kubernetes-kubeminions'].0 }}"
   become: true
 
+- name: Get $HOME of K8s master for kubernetes user
+  shell: source ~/.profile; echo $HOME
+  args:
+    executable: /bin/bash
+  register: result_kube_home
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
```

Comment:
how will this work in a setup like GKE where there is no access to master?

Reply:
In GKE the Kubemaster and Localhost will refer to the same IPs. All the logs are stored on the Localhost.